.github/workflows/zap_templates.yaml

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) 2020 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: ZAP
+
+on:
+    push:
+    pull_request:
+
+jobs:
+    templates:
+        name: ZAP templates generation
+
+        runs-on: ubuntu-latest
+
+        strategy:
+           matrix:
+            node-version: [12.x]
```

Comment:
do we need a matrix? It seems like a single run is needed and then we can just hardcode the node version without a matrix of settings.

Reply:
Removed